### PR TITLE
test-configs.yaml: enable all current LTP subsets on coral

### PR DESCRIPTION
**Pull request by gctucker**
Enable a few missing LTP subsets on asus-C523NA-A20057-coral to cover
all the ones currently defined.

Signed-off-by: Guillaume Tucker <guillaume.tucker@collabora.com>

### CODE REVIEW

**Comment by gctucker** [comment]
This hasn't been deployed on staging yet as it's conflicting with #715.  Waiting until it's merged for now.

File: config/core/test-configs.yaml
Comment (gctucker) [inline config/core/test-configs.yaml:1743]: Tested OK on staging https://staging.kernelci.org/test/plan/id/60eea91585767628e1cef2d6/

File: config/core/test-configs.yaml
Comment (gctucker) [inline config/core/test-configs.yaml:1747]: Tested OK on staging https://staging.kernelci.org/test/plan/id/60eea837a66a7d86a4cef2eb/

File: config/core/test-configs.yaml
Comment (gctucker) [inline config/core/test-configs.yaml:1748]: Tested OK on staging https://staging.kernelci.org/test/plan/id/60eea4fd7ad84b846bcef304/

**Review by alexandrasp — APPROVED** [review]
LGTM

**Pull request merged**
